src/session/content/item_row.rs

```diff
@@ -94,16 +98,290 @@ impl ItemRow {
+                                                    } else {
+                                                        // Translators: This string is used to separate names of two users e.g. Tom and Jerry
+                                                        Cow::Owned(gettext!(
+                                                            "{} and {}",
+                                                            // Translators: This comma is used to separete names of two users
+                                                            first_members.join(&gettext(", ")),
+                                                            last_member
+                                                        ))
+                                                    }
+                                                )
+                                            } else {
+                                                gettext!(
+                                                    "{} added {} to the group",
+                                                    sender_name,
+                                                    if first_members.is_empty() {
+                                                        Cow::Borrowed(last_member)
+                                                    } else if remove_you(
+                                                        data.clone().member_user_ids,
+                                                        my_user_id
+                                                    ) != data.member_user_ids
+                                                    {
+                                                        // Translators: This string is used to separate names of two users e.g. Tom and Jerry
+                                                        Cow::Owned(gettext!(
+                                                            "{} and you",
+                                                            // Translators: This comma is used to separete names of two users
+                                                            members.join(&gettext(", ")),
+                                                        ))
+                                                    } else {
+                                                        // Translators: This string is used to separate names of two users e.g. Tom and Jerry
+                                                        Cow::Owned(gettext!(
+                                                            "{} and {}",
+                                                            // Translators: This comma is used to separete names of two users
+                                                            first_members.join(&gettext(", ")),
+                                                            last_member
+                                                        ))
+                                                    }
```

Comment:
1. You build the members `Vec<_>` again by calling `remove_you`. But I think this is not needed, as we still could use that `Vec` from the lines above.
2. You fully compare both Vecs. I think this is an overkill because we only need to compare the length of the Vecs.
3. I would use just one `Cow::Owned` by bringing it the outer scope.    

Here is my suggestion
```suggestion
                                                   } else {
                                                        Cow::Owned(
                                                            if data.member_user_ids.len()
                                                                != members.len()
                                                            {
                                                                // Translators: This string is used to separate names of two users e.g. Tom and Jerry
                                                                gettext!(
                                                                    "{} and you",
                                                                    // Translators: This comma is used to separete names of two users
                                                                    members.join(&gettext(", ")),
                                                                )
                                                            } else {
                                                                // Translators: This string is used to separate names of two users e.g. Tom and Jerry
                                                                gettext!(
                                                                    "{} and {}",
                                                                    // Translators: This comma is used to separete names of two users
                                                                    first_members
                                                                        .join(&gettext(", ")),
                                                                    last_member
                                                                )
                                                            },
                                                        )
                                                    }
```